### ISSUE TITLE
External links

### ISSUE THREAD

**Issue by JelteMX** [mendix/docs#93]
Report from external link checker (Gulp):

```text
File: /refguide5/consumed-web-services.html has the following errors:

   + Has link to http://www.w3schools.com/webservices/tempconvert.asmx?WSDL which return a Page not found. Please fix this

File: /howto6/access-a-samba-share-from-the-mxcloud.html has the following errors:

   + Has link to http://www.stderr.org/doc/libjcifs-java-doc/resolver.html which return a Page not found. Please fix this
   + Has link to http://www.stderr.org/doc/libjcifs-java-doc/wins.html which return a Page not found. Please fix this

File: /howto6/deploy-a-mendix-app-to-ibm-bluemix.html has the following errors:

   + Has link to https://api.ng.bluemix.net/ which return a Page not found. Please fix this
   + Has link to https://api.eu-gb.bluemix.net which return a Page not found. Please fix this
   + Has link to https://api.au-syd.bluemix.net which return a Page not found. Please fix this

File: /howto50/access-a-samba-share-from-the-mxcloud.html has the following errors:

   + Has link to http://www.stderr.org/doc/libjcifs-java-doc/resolver.html which return a Page not found. Please fix this
   + Has link to http://www.stderr.org/doc/libjcifs-java-doc/wins.html which return a Page not found. Please fix this
   + Has link to http://www.stderr.org/doc/libjcifs-java-doc/overview.html#scp which return a Page not found. Please fix this

```

These links should be checked and/or fixed

**Comment by Adam-Dupaski** [comment]
Fixed file #2. Waiting on verification for possible fixes for other files.

**Comment by JelteMX** [comment]
Fixed all the links. The **bluemix.net** links are example links, so these can be ignored
